inc/class-saml.php

```diff
@@ -954,15 +964,17 @@ public function createUser( $username, $email ) {
 	 *
 	 * @return string
 	 */
-	public function sanitizeUser( $username ) {
+	public function sanitizeUser( string $username ): string {
+		$username = substr( $username, 0, 59 );
```

Comment:
I used one of the test providers (Mock SAML) and it sends larger `uid` (> 60 characters). It might happen with another provider causing to fail to create the user.